### PR TITLE
Add test "CRUD multistore"

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add test "CRUD multistore"
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/14_advancedParameters/10_multistore/01_CRUDShopGroups.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21973)

### CODE REVIEW

File: tests/UI/campaigns/data/faker/shopGroup.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const faker = require('faker');
+
+module.exports = class SeoPage {
```
Comment (SimonGrn) [inline tests/UI/campaigns/data/faker/shopGroup.js]:
```suggestion
module.exports = class ShopGroup {
```

Comment (nesrineabdmouleh) [reply tests/UI/campaigns/data/faker/shopGroup.js]: Done thx :+1:

File: tests/UI/campaigns/functional/BO/14_advancedParameters/10_multistore/01_CRUDShopGroups.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+const multiStorePage = require('@pages/BO/advancedParameters/multistore');
+const addShopGroupPage = require('@pages/BO/advancedParameters/multistore/add');
+
+// Import data
+const ShopGroupFaker = require('@data/faker/shopGroup');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_modules_advancedParameters_webservice_CRUDShopGroups';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/10_multistore/01_CRUDShopGroups.js]:
```suggestion
const baseContext = 'functional_BO_modules_advancedParameters_multistore_CRUDShopGroups';
```

Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/14_advancedParameters/10_multistore/01_CRUDShopGroups.js]: Done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Pull request merged**